.storybook/preview.js

```diff
@@ -22,6 +22,44 @@ configureActions({
   limit: 20,
 });
 
+const customViewports = {
+  ExtraSmall: {
```

Comment:
Normally object keys would start with a lower case letter (camelCase) rather than SentenceCase - not the end of the world though